stylesheets/style.css

```diff
@@ -0,0 +1,71 @@
+body {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    display: grid;
+    grid-template-rows: 175px 1fr 60px;
+    background-color: lavenderblush;
```

Comment:
Style: I know we do this in class but in general it's best practices not to use these names in production websites (it's fine for assignments, just something to keep in mind for industry).